### ISSUE TITLE
learn.summary() crashes out on 2nd transfer learning

### ISSUE THREAD

**Issue by ai-padawan** [fastai/fastai#2735]
First reported bug here:
https://forums.fast.ai/t/bug-learn-summary-does-not-work-on-2nd-transfer-learning/77897

**tldr:** learn.summary() crashes out with the following summary when doing a 2nd cycle of transfer learning
```
 57         elif val <= self.first_its or val >= self.last_v + self.wait_for or val >= self.total:
 58             cur_t = time.time()
 59             avg_t = (cur_t - self.start_t) / val
 60             self.wait_for = max(int(self.update_every / (avg_t+1e-8)),1)
 61             self.pred_t = avg_t * self.total
 
 AttributeError: 'NBProgressBar' object has no attribute 'start_t'
```

When I revert back the fastai code (in the fastai2 repo to commit: 59d878d3cf233ea24eb8fd8987098f17edd8c8ef) this crash goes away. I've isolated it to the subsequent commit d9ed4a8337bab36d3680fd787494e83ebd2f9a4b, with refactor learn.summary() that causes this error.

**Comment by raphaelr** [comment]
This also breks `learn.tta()`

**Comment by jph00** [comment]
@ai-padawan , as mentioned in the issue template, we need full details on how to repro the bug, in order to fix it.

Can you please provide a complete code example which triggers this error?

**Comment by jph00** [comment]
I suspect my fix to #2764 should fix this too. Feel free to reopen if it's not fixed.
